feature-group-definitions/fullscreen.yml

```
@@ -1,4 +1,5 @@
 name: Fullscreen API
+description: The fullscreen API is used to present a specific element in fullscreen mode.
```

Avoiding a circular description

```suggestion
description: The fullscreen API makes a specific element fill the whole screen and hides most browser UI.
```